.github/linters/action.yaml

```diff
@@ -0,0 +1,42 @@
+name: 'Linters'
+outputs:
+  mypy:
+    description: "Mypy Result"
+    value: ${{ steps.set-outputs.outputs.mypy }}
+  black:
+    description: "Black Result"
+    value: ${{ steps.set-outputs.outputs.black }}
+  flake8:
+    description: "Flake8 Result"
+    value: ${{ steps.set-outputs.outputs.flake8 }}
+runs:
+  using: "composite"
+  steps:
+      - name: Mypy linting
+        id: mypy
```

Comment:
this doesn't change the mypy config we were using before in any way, correct?  just tweaks how we are launching it?

Reply:
Yep, there's a `mypy.ini` that wasn't modified in any way.